aggregating-profiler-prometheus/build.gradle.kts

```diff
@@ -10,15 +7,14 @@ dependencies {
     /**
      * Runtime
      */
-    compile(Libs.slf4j_api)
-
-    compile(project(":aggregating-profiler"))
+    api(Libs.slf4j_api)
```

Comment:
![image](https://user-images.githubusercontent.com/9992425/83111411-379d0a80-a0cd-11ea-8f1a-4168309dfde0.png)
Actually, I don't see any usages of logging libraries in aggregating-profiler-prometheus

Reply:
I removed that dependency